schemas/src/enums/InputLabelPosition.json

```diff
@@ -0,0 +1,9 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+	"classType": "Enum",
+	"description": "Specifies position of the label with respect to input.",
+	"values": [
+		"inline",
+		"above"
```

Comment:
Aren't we adding `default` as an option here as well?

Reply:
"above" is the default behavior